.github/workflows/tests.yml

```diff
@@ -41,6 +42,25 @@ jobs:
         ./test/wait-for-hawkbit-online
         cd test && ./rauc-hawkbit-updater.t -v
 
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install required packages
+      run: |
+        sudo apt-get update
+        sudo apt-get install libcurl4-openssl-dev libsystemd-dev libjson-glib-dev python-sphinx doxygen
```

Comment:
`libsystemd-dev` shouldn't be needed here.

Reply:
Ack, removed